src/test/clip.t.sol

```diff
@@ -1298,6 +1323,23 @@ contract ClipperTest is DSTest {
         assertTrue(try_redo(1, address(this)));
     }
 
+    function testFail_stopped_3_auction_reset_tail() public {
+        clip.file("stopped", 3);
+
+        auctionResetSetup(10 hours); // 10 hours till zero is reached (used to test tail)
+
+        pip.poke(bytes32(uint256(3 ether))); // Spot = $1.50 (update price before reset is called)
+
+        (,,,, uint96 ticBefore, uint256 topBefore) = clip.sales(1);
+        assertEq(uint256(ticBefore), startTime);
+        assertEq(topBefore, ray(5 ether)); // $4 spot + 25% buffer = $5 (wasn't affected by poke)
+
+        hevm.warp(startTime + 3600 seconds);
+        assertTrue(!try_redo(1, address(this)));
+        hevm.warp(startTime + 3601 seconds);
+        assertTrue(try_redo(1, address(this)));
```

Comment:
I'm not really sure I'm following these tests, they are `testFail` but we are using the `try` type of functions which would allow us to avoid them. We can better remove the `Fail` and just do `assertTrue(!try_redo,...`
Same for the prev test.

Btw, this kind of test would be even better with a `try` function which actually only tests if gonna succeed without actually keeping the change. In `rwa-example` they were implemented. Basically you need some wrapper that executes the change via a `call`, checks it fails or not, and if it doesn't fail make a `revert`.

Reply:
Hmmmm yeah something is weird about these, I didn't pay too close attention and just followed the pattern...

Reply:
Okay fixed the issue...classic example of why `testFail` is an antipattern except in extremely simple cases.